src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshaling/JSMarshalerArgument.Task.cs

```diff
@@ -57,10 +59,16 @@ public unsafe void ToManaged(out Task? value)
                 if (arg_2.slot.Type != MarshalerType.None)
                 {
                     arg_2.ToManaged(out Exception? fail);
+#if FEATURE_WASM_THREADS
+                    JSProxyContext.PopOperation();
```

Comment:
This is surprising to me. I can kind of understand why it's here, but updating the xmldoc comment to mention that it pops the operation would be good.

Reply:
added comment